skore/src/skore/_config.py

```diff
@@ -61,6 +69,13 @@ def set_config(
     show_progress : bool, default=None
         If True, show progress bars. Otherwise, do not show them.
 
+    plot_backend : {"matplotlib", "plotly"}, default="matplotlib"
```

Comment:
```suggestion
    plot_backend : {"matplotlib", "plotly"}, default=None
```

I don't think it is the right place to mention that the default backend is `matplotlib`.